config.js

```diff
@@ -6,6 +6,7 @@
 
 const env = process.env.NODE_ENV;
 const useMocks = process.env.USE_MOCKS === 'true' || !env;
+const port = process.env.PORT || 8080;
```

Comment:
can we remove the defaults of 8080 please, it's considered bad practice to use defaults?

Reply:
I think this was an old change it should already be removed